webpack.config.babel.js

```diff
@@ -91,6 +91,9 @@ export default env => {
     // in base.js are more general
     'resolve.alias': 'prepend',
   })(webpackDefaults(env, config), {
+    node: {
+      fs: 'empty',
```

Comment:
what does this do (and why is it needed)? might want to leave a commend here since it's not straightforward

Reply:
Sounds like some naughty lib tries to use node's `fs` module.

Reply:
(BTW, this just tells webpack to mock it, IIRC)

Reply:
So, the problem is with the addition of `micromatch`. I think some of its dependencies uses `fs` and this says to mock the module entirely (as @pferreir pointed out correctly)